style.md

```diff
@@ -2837,6 +2838,54 @@ tests := []struct{
 }
 ```
 
+#### Omit Zero Value Fields in Structs
+
+When initializing structs with field names, omit fields that have zero values.
```

Comment:
rather than calling out the exception, can we incorporate it into the recommendation, "omit fields that have zero values where the field doesn't provide meaningful context" or something like that?

Reply:
Agree, despite calling it out below:

> This helps reduce noise for readers by omitting values that are default in
that context. Only meaningful values are specified.

```suggestion
When initializing structs with field names, omit fields that have zero values unless they provide meaningful context.
```

Reply:
Good point. Addressed.